### PR TITLE
Added Rusin language input layout.

### PR DESCRIPTION
**Pull request by novta**
Hello,

I have developed Rusin language input layout for region Serbia. I would like to publish it over your channel. I would take responsibility to contribute further support on Rusin keyboard layout.
I read that i can do it independently but I see to do it over you as more appropriate.
I intend to develop dictionary in future if there will be request from my users. My blog about keyboards i have supported is here http://novta.blogspot.com/. 

Thank you,
Miroslav Novta

### CODE REVIEW

File: gradle/language_pack_lib.gradle
Comment (menny) [inline gradle/language_pack_lib.gradle]: This needs to be simplified:
```
def dimensArray = [['xxxhdpi', 96, 80], ['xxhdpi', 72, 62], ['xhdpi', 48, 40], ['hdpi', 36, 32], ['mdpi', 24, 22]]
if(project.ext.status_icon_text.length() == 3) {
    dimensArray = [['xxxhdpi', 96, 64], ['xxhdpi', 72, 50], ['xhdpi', 48, 32], ['hdpi', 36, 24], ['mdpi', 24, 16]]
}

dimensArray.each { dimens ->

```

Comment (menny) [reply gradle/language_pack_lib.gradle]: and then the rest of the calls. Without duplicating the tasks registration part.

File: languages/rusin/apk/build.gradle
Comment (menny) [inline languages/rusin/apk/build.gradle]: So, to be clear, this pack should be separate from Serbian?

Comment (menny) [reply languages/rusin/apk/build.gradle]: also, Wikipedia says it should be `Rusyn`: https://en.wikipedia.org/wiki/Rusyn_language
Is that the English spelling and `rusin` is the native spelling?

File: languages/rusin/apk/src/main/res/values/strings.xml
Comment (menny) [inline languages/rusin/apk/src/main/res/values/strings.xml]: This should be in the native language.

File: languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java
Comment (menny) [inline languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java]: Did you moved this file from the Serbian pack?

**Review by menny — CHANGES_REQUESTED** [review]
Thanks. I have a few needed clarifications

File: languages/rusin/apk/build.gradle
Comment (novta) [reply languages/rusin/apk/build.gradle]: Language is Rusyn, spelled in English. Should it be called in English?

File: languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java
Comment (novta) [reply languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java]: Yes, it is based on Serbian keyboard leyout. 
Layout is for Rusyn language for Serbia as country. 
We have same kaybord, the majority of keys are at same position, there are some modifications.

File: languages/rusin/apk/build.gradle
Comment (novta) [reply languages/rusin/apk/build.gradle]: Country of origin is Serbia but language is different that Serbian.

File: gradle/language_pack_lib.gradle
Comment (novta) [reply gradle/language_pack_lib.gradle]: it is fixed

File: languages/rusin/apk/build.gradle
Comment (novta) [reply languages/rusin/apk/build.gradle]: it is fixed to be Rusyn everywhere

File: languages/rusin/apk/src/main/res/values/strings.xml
Comment (novta) [reply languages/rusin/apk/src/main/res/values/strings.xml]: It is fixed to be Rusyn for AnySoftKeyboard.
I see other languages like Russian and Belarus did not used cyrilic letters too so.

File: languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java
Comment (novta) [reply languages/rusin/pack/src/main/java/com/anysoftkeyboard/languagepack/rusin/PackBroadcastReceiver.java]: The ties to Serbian dictionary are removed. there should not be and tracable connection.

File: languages/rusyn/pack/src/main/java/com/anysoftkeyboard/languagepack/rusyn/PackBroadcastReceiver.java
Comment (menny) [inline languages/rusyn/pack/src/main/java/com/anysoftkeyboard/languagepack/rusyn/PackBroadcastReceiver.java]: you moved this file to rusyn location, instead of copying it.

**Review by menny — CHANGES_REQUESTED** [review]

File: languages/rusyn/pack/src/main/java/com/anysoftkeyboard/languagepack/rusyn/PackBroadcastReceiver.java
Comment (novta) [reply languages/rusyn/pack/src/main/java/com/anysoftkeyboard/languagepack/rusyn/PackBroadcastReceiver.java]: Yes, It is copied from after successfull build serbian to rusin and then renamed to rusyn. 
Is this a file that is to be generated durring build?

Comment (menny) [reply languages/rusyn/pack/src/main/java/com/anysoftkeyboard/languagepack/rusyn/PackBroadcastReceiver.java]: I don't see the Serbian file anymore. Please make sure you have both files on your branch. I can't merge this PR if it breaks other packs.

Comment (novta) [reply languages/rusyn/pack/src/main/java/com/anysoftkeyboard/languagepack/rusyn/PackBroadcastReceiver.java]: Unwanted change in serbian is reverted.

**Comment by menny** [comment]
The good news is that now this PR is ready. thanks.

The bad news is that this repository is out of commission. We now use https://github.com/AnySoftKeyboard/AnySoftKeyboard/tree/master/addons/languages for language-pack source-control.

**Comment by menny** [comment]
would you mind re-creating this PR there?

**Comment by novta** [comment]
I have done it, PR link is here https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2002

**Pull request closed without merging**
